zebrad/tests/acceptance.rs

```diff
@@ -1706,6 +1706,57 @@ async fn delete_old_databases() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn stored_config_works() -> Result<()> {
+    zebra_test::init();
+
+    let run_dir = testdir()?;
+    let stored_config_path = stored_config_path();
+
+    // run zebra with stored config
+    let mut child =
+        run_dir.spawn_child(args!["-c", stored_config_path.to_str().unwrap(), "start"])?;
+
+    //zebra was able to start with the stored config
+    child.expect_stdout_line_matches("Starting zebrad".to_string())?;
+
+    // finish
+    child.kill()?;
+
+    let output = child.wait_with_output()?;
+    let output = output.assert_failure()?;
+
+    // [Note on port conflict](#Note on port conflict)
+    output
+        .assert_was_killed()
+        .wrap_err("Possible port conflict. Are there other acceptance tests running?")?;
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn stored_config_is_newest() -> Result<()> {
```

Comment:
I don't think this needs to be an async function.

Reply:
Thanks, they don't need to be async at all. https://github.com/ZcashFoundation/zebra/pull/4676/commits/2483cea8a02eacebcba3703f0bb2b1573f92effc